ecommerce/frontend/src/components/SearchInput.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import './SearchInput.less'
+import { Select } from 'antd'
+import Search from 'antd/lib/input/Search'
```

Comment:
This is the wrong import (not designed for public use). I believe you can import it alongside the `{ Select }` import.